src/main/java/Task.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * The Task class represents the things to do.
+ */
+public class Task {
```

Comment:
Since the Task class is primarily used for inheritance for other subclasses, and we do not actually save a Task parent object, its better to make the class abstract instead.